modules/supported-log-outputs.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * logging/log_collection_forwarding/logging-output-types.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="supported-log-outputs_{context}"]
+= Supported log forwarding outputs
+
+Outputs can be any of the following types:
+
+.Supported log output types
+[cols="5",options="header"]
+|===
+|Output type
+|Protocol
+|Tested with
+|Logging versions
+|Supported collector type
+
+|Elasticsearch v6
+|
+|6.8.1, 6.8.23
+|5.8, 5.7, 5.6
+|Fluentd, Vector
+
+|Elasticsearch v7
+|
+|7.12.2, 7.17.7, 7.10.1
+|5.8, 5.7, 5.6, 5.5
+|Fluentd, Vector
+
+|Elasticsearch v8
+|
+|8.4.3, 8.6.1
+|5.8, 5.7, 5.6
+|Fluentd ^[1]^, Vector
+
+|Fluent Forward
+|Fluentd forward v1
+|Fluentd 1.14.6, Logstash 7.10.1, Fluentd 1.14.5
+|5.8, 5.7, 5.6, 5.5, 5.4
+|Fluentd
+
+|Google Cloud Logging
+|
+|
```

Comment:
For google cloud and cloudwatch, I think stating "Latest" may be sufficient??   Their services are constantly updated.  The other option would be to specify the api version we are using to interact.